downloadDirectory/pom.xml

```diff
@@ -0,0 +1,181 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.reactome.release</groupId>
+  <artifactId>downloadDirectory</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>downloadDirectory</name>
+  <url>http://maven.apache.org</url>
```

Comment:
I think the `url` element is supposed to be for the project's URL, so you could put the github repo URL for download-directory, or just remove this element (I'm pretty sure it's optional).